src/variations/frankenphp/Dockerfile

```diff
@@ -0,0 +1,228 @@
+    echo "${FRANKENPHP_VERSION}" > /etc/serversideup-php-frankenphp-version;
+    # \
+    # # Make composer cache directory
+    # mkdir -p "${COMPOSER_HOME}"; \
+    # chown -R www-data:www-data "${COMPOSER_HOME}";
+
+####################
+# Go Image
+####################
+FROM golang:${GOLANG_VERSION} AS golang-image
+
+####################
+# FrankenPHP Build
+####################
+FROM common AS frankenphp-build
+ARG FRANKENPHP_VERSION
+ARG GOLANG_VERSION
+ARG BUILD_DEPENDENCY_PACKAGES_ALPINE='argon2-dev bash brotli-dev ca-certificates coreutils curl-dev git gnu-libiconv-dev libcap libsodium-dev cmake libstdc++ libxml2-dev linux-headers mailcap oniguruma-dev openssl-dev readline-dev sqlite-dev upx'
+ARG BUILD_DEPENDENCY_PACKAGES_DEBIAN='cmake git libargon2-dev libbrotli-dev libcap2-bin libcurl4-openssl-dev libonig-dev libreadline-dev libsodium-dev libsqlite3-dev libssl-dev libxml2-dev mailcap zlib1g-dev'
+
+COPY --from=golang-image /usr/local/go /usr/local/go
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV GOTOOLCHAIN="local"
+
+# Copy xcaddy in the builder image
+COPY --from=caddy:builder /usr/bin/xcaddy /usr/bin/xcaddy
+
+# Install dependencies & Download FrankenPHP
+RUN docker-php-serversideup-dep-install-alpine "$PHPIZE_DEPS ${BUILD_DEPENDENCY_PACKAGES_ALPINE}" && \
+    docker-php-serversideup-dep-install-debian "${BUILD_DEPENDENCY_PACKAGES_DEBIAN}"
+
+# Install e-dant/watcher (necessary for file watching)
+WORKDIR /usr/local/src/watcher
+RUN curl -s https://api.github.com/repos/e-dant/watcher/releases/latest | \
+    grep tarball_url | \
+    awk '{ print $2 }' | \
+    sed 's/,$//' | \
+    sed 's/"//g' | \
+    xargs curl -L | \
+    tar xz --strip-components 1 && \
+    cmake -S . -B build -DCMAKE_BUILD_TYPE=Release && \
+    cmake --build build && \
+    cmake --install build && \
+    if cat /etc/os-release | grep -q 'debian'; then \
+        ldconfig; \
+    fi
+
+# Download and build FrankenPHP
+WORKDIR /go/src/app
+ENV GOBIN=/usr/local/bin
+
+RUN if cat /etc/os-release | grep -q 'debian'; then \
+        export ADDITIONAL_BUILD_FLAGS=''; \
+    elif cat /etc/os-release | grep -q 'alpine'; then \
+        export ADDITIONAL_BUILD_FLAGS="-extldflags '-Wl,-z,stack-size=0x80000'"; \
+    fi; \
+    git clone --depth 1 --branch v${FRANKENPHP_VERSION} \
+    https://github.com/php/frankenphp.git .; \
+    CGO_ENABLED=1 \
+    XCADDY_SETCAP=1 \
+    XCADDY_GO_BUILD_FLAGS="-ldflags='-w -s' -tags=nobadger,nomysql,nopgx" \
+    CGO_CFLAGS="$(php-config --includes) $ADDITIONAL_BUILD_FLAGS" \
+    CGO_LDFLAGS="$(php-config --ldflags) $(php-config --libs)" \
+    xcaddy build \
+        --output /usr/local/bin/frankenphp \
+        --with github.com/dunglas/frankenphp=./ \
+        --with github.com/dunglas/frankenphp/caddy=./caddy/ \
+        --with github.com/dunglas/caddy-cbrotli \
+        # Mercure and Vulcain are included in the official build, but feel free to remove them
```

Comment:
Could you install this by default to give people the option of an alternative logging format rather than the caddy default which is not really designed for humans.
https://caddyserver.com/docs/modules/caddy.logging.encoders.transform